docs/contributing/authoring-polyfills.html

```diff
@@ -0,0 +1,173 @@
+
+		<p>Objects defined as part of ECMAScript and present in all browsers above our baseline may be assumed to be present, so this is fine:</p>
+
+		<pre><code>'imul' in Math</code></pre>
+
+		<h2 id='update-tasks'>Update tasks for 3rd party polyfills</h2>
+
+		<p>If a polyfill comes from a third party source, it's a good idea to write an update task that can be run to import the latest version of the polyfill code from its master repo.  An update task is called <code>update.task.js</code> and lives alongside the <code>polyfill.js</code> and <code>config.json</code> files. It must export a function that accepts the <code>grunt</code> object, and currently must complete its work synchronously.  Example:</p>
+
+		<pre><code>'use strict';
+var path = require('path');
+module.exports = function(grunt) {
+	// ... download/import latest source here
+	// ... do any required source modification
+	var newSource = 'test';
+	grunt.file.write(path.join(__dirname, 'polyfill.js'), newSource);
+};</code></pre>
+
+		<p>Often changes are required (or desirable) to support the format of polyfills needed by the polyfill service, for example:</p>
+
+		<ul>
+		<li><strong>Private dependencies:</strong> some polyfill authors will create polyfill collections that abstract common tasks into shared functions that are not themselves public web platform APIs.  The polyfill service supports this via private polyfills (which are prefixed <code>_</code>), and will import and declare them in the parent scope so that the polyfill can consider them to be globals.  Polyfills may therefore need to be amended to remove import or require statements (or to provide an import or require implementation that will satisfy them), and to provide the necessary dependency via other means (either as a seaparate private polyfill or by inlining it)</li>
+		<li><strong>Embedded polyfills:</strong> some polyfills include code that fills more than one feature. For example, <code>classList</code> polyfills typically also polyfill <code>DOMTokenList</code>.  This should be a dependency relationship expressed using the <code>config.json</code> metadata, rather than conflating two polyfills into one.</li>
+		<li><strong>Feature detects / guards:</strong> most polyfills contain feature detects to ensure that they do not take effect if the feature is already present.  Ours do not, because the feature detect is stored separately in <code>detect.js</code> and combined with the polyfill source at runtime.  If possible, a polyfill should be imported without embedded detects.</li>
+		</ul>
+
+		<h2 id="browsers">Browsers</h2>
+
+		<p>The short names should be used in the <code>config.json</code> to configure the browser support using the <code>browsers</code> key.</p>
+
+		<table>
+			<thead>
+			<tr><th>Short name</th><th>User Agent Name</th><th>Baseline</th></tr>
+			</thead>
+			<tbody>
+			<tr><td><code>ie</code></td><td>Internet Explorer / Edge / Edge Mobile</td><td>{{baselines.ie}}</td></tr>
```

Comment:
The baseline data isn't showing on on the QA site, not sure if this is environment issue or an issue caused by this pull request